dbtvault-dev/dbt_project.yml

```diff
@@ -17,5 +17,6 @@ clean-targets:
 
 vars:
   hash: MD5
+  hash_case: 'UNCHANGED' # Default UPPER, alternatively UNCHANGED
```

Comment:
We should make the default UPPER as otherwise anyone running dbtvault is going to get a nasty surprise when their hashes suddenly get re-generated.